meta/FlexibleSUSY.m

```diff
@@ -207,14 +207,6 @@ FlexibleSUSY model file (FlexibleSUSY.m).
 this List is rewritten during the runnig of start.m script
 in the directory FlexibleSUSY/models/@CLASSNAME@ by
 the Get[FlexibleSUSY.m] inside FlexibleSUSY`MakeFlexibleSUSY[ ... ]";
-ExtraSLHAOutputBlocks = {
```

Comment:
We're deactivating the calculation of the amm by default now, right?

Reply:
Yes. Unless I shouldn't. Anyway, the way it was written didn't work for `NoFV` models.

Reply:
We can leave it like this I think (especially if it did not work correctly for all models).